cmd/kops/get_federation.go

```diff
@@ -82,19 +84,21 @@ func RunGetFederations(context Factory, out io.Writer, options *GetFederationOpt
 			return strings.Join(f.Spec.Members, ",")
 		})
 		return t.Render(federations, out, "NAME", "CONTROLLERS", "MEMBERS")
-	} else if output == OutputYaml {
+
+	case OutputYaml:
```

Comment:
Oh ... idea: We could declare a function that maps OutputYaml -> marshalYaml etc.  We could even make OutputYaml a named type (`type OutputFormat string`) and then declare a function on `OutputFormat` (I think)